lib/logstash/inputs/stomp.rb

```diff
@@ -59,6 +75,10 @@ def register
       connect
       subscription_handler # is required for re-subscribing to the destination
     }
+
+    # Display configuration information
+    @logger.info("Reconnect is disabled, plugin will not reconnect on connection error") if !@reconnect
+    @logger.info("Debug is disabled, turn on debug for full exceptions") if !@debug
```

Comment:
Thanks for the information. I played around a little with the --verbose and --debug flags. And adjusted the logging accordingly.